src/server/transaction.h

```diff
@@ -129,11 +129,12 @@ class Transaction {
   enum LocalMask : uint16_t {
     ACTIVE = 1,  // Set on all active shards.
     // UNUSED = 1 << 1,
-    OUT_OF_ORDER = 1 << 2,      // Whether its running out of order
+    OUT_OF_ORDER = 1 << 2,      // Whether it can run as out of order
     KEYLOCK_ACQUIRED = 1 << 3,  // Whether its key locks are acquired
     SUSPENDED_Q = 1 << 4,       // Whether is suspened (by WatchInShard())
     AWAKED_Q = 1 << 5,          // Whether it was awakened (by NotifySuspended())
     EXPIRED_Q = 1 << 6,         // Whether it timed out and should be dropped
+    UNLOCK_MULTI = 1 << 7,      // Whether this shard executed UnlockMultiShardCb
```

Comment:
what is the value of this enum ? I see you just added a place that you set the bit, but I dont see checks on it

Reply:
for debugging only